### PR TITLE
work completed

### PR DESCRIPTION
**Pull request by Henshall**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env
+.env.development.local
```
Comment (Henshall) [inline .gitignore:17]: these files dont exist please remove from git ignore

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Follow setup to run app:
+
+Make sure you have nvm installed
+NPM version 6.14.8
+
+create .env in project
+Update in .env REACT_APP_WEATHER_API_KEY = "YOUR-KEY"
```
Comment (Henshall) [inline README.md]: this doesnt look good on github. we need bullet points and steps

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  }, [address]);
+
+  const getLocationLatLong = (lat, lang) => {
+    Geocode.fromLatLng(lat, lang).then(
+      (response) => {
+        const address = response.results[0].formatted_address;
+        let city, state, country;
+        for (
+          let i = 0;
+          i < response.results[0].address_components.length;
```
Comment (Henshall) [inline src/App.js:82]: we should put all of these on one  line... formatting is really bad

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    <div className="App">
+      <header className="App-header pt-5">
+        <div className="container">
+          <div className="wrapper pt-5">
+            <div className="row">
+              <div className="col-2"></div>
+              <div className="col-8">
+                <div className="input-group">
+                  <FaCrosshairs
+                    onClick={getLocation}
```
Comment (Henshall) [inline src/App.js:121]: formatting one line

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+                    className="icon-location"
+                  />
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="Enter Location..."
```
Comment (Henshall) [inline src/App.js:127]: formatting one line

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+                  <div className="col-sm-12 col-xs-12 col-md-12 col-lg-6 col-xl-6 pt-5">
+                    <h1 className="teal-color big-text">
+                      {kelvinToFahrenheit(locationInfo.main.temp)}&deg; C
+                    </h1>
+                    <br />
+                    <ReactCountryFlag
+                      countryCode={locationInfo.sys.country}
+                      style={{
+                        fontSize: "2em",
+                        lineHeight: "3em",
```
Comment (Henshall) [inline src/App.js:147]: no styles in html doc. also formattting

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+                      <FaLocationArrow /> {locationInfo.name}
+                    </h2>
+                    <br />
+                    <h5>{countryData.country}</h5>
+                  </div>
+                  <div className="col-sm-12 col-xs-12 col-md-12 col-lg-6 col-xl-6 pt-5">
+                    <div className="weatherWrapper">
+                      <img
+                        src={`http://openweathermap.org/img/w/${locationInfo?.weather[0].icon}.png`}
+                        alt="weather status icon"
```
Comment (Henshall) [inline src/App.js:162]: formatting one line

File: src/App.test.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```
Comment (Henshall) [inline src/App.test.js]: why do we have a test file?

File: yarn.lock [hunk 9]
```diff
@@ -0,0 +1,11465 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (Henshall) [inline yarn.lock:2]: is this file needed?

**Review by Henshall — COMMENTED** [review]
please fix changes

File: yarn.lock (hunk 9 above)
Comment (FullStackkDev) [reply yarn.lock:2]: yes this file is required

File: .gitignore (hunk 1 above)
Comment (FullStackkDev) [reply .gitignore:17]: if someone is working on mac platform or add these files in future. So best practice is to add these files in gitignore. 

Its standard practice

File: src/App.js (hunk 3 above)
Comment (FullStackkDev) [reply src/App.js:82]: @Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/

File: src/App.js (hunk 4 above)
Comment (FullStackkDev) [reply src/App.js:121]: @Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/

File: src/App.js (hunk 5 above)
Comment (FullStackkDev) [reply src/App.js:127]: @Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/

File: src/App.js (hunk 6 above)
Comment (FullStackkDev) [reply src/App.js:147]: @Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/

File: src/App.js (hunk 7 above)
Comment (FullStackkDev) [reply src/App.js:162]: @Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/

File: README.md (hunk 2 above)
Comment (FullStackkDev) [reply README.md]: @Henshall done

File: src/App.test.js (hunk 8 above)
Comment (FullStackkDev) [reply src/App.test.js]: @Henshall test file has been removed

File: src/App.js (hunk 3 above)
Comment (Henshall) [reply src/App.js:82]: okay
